src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalc, Draw {
```

Comment:
The class 'Figure' should not be abstract if it is used to set behavior for classes. Consider using an interface to define behavior or a concrete class if you need to share common state or functionality.